glusterd2/options/cluster.go

```diff
@@ -38,6 +38,7 @@ var ClusterOptMap = map[string]*ClusterOption{
 	"auto-create-block-hosting-volumes":  {"auto-create-block-hosting-volumes", "true", OptionTypeBool, nil},
 	"block-hosting-volume-replica-count": {"block-hosting-volume-replica-count", "3", OptionTypeInt, nil},
 	"block-hosting-volume-type":          {"block-hosting-volume-type", "Replicate", OptionTypeStr, nil},
+	"auto-delete-block-hosting-volumes":  {"auto-delete-block-hosting-volumes", "true", OptionTypeBool, nil},
```

Comment:
I think it's better to make default as false or delete BHV if more than one are there, this avoids the frequent creation and deletion of BHV if we keep on creating and deleting single PVC


@aravindavk any thoughts?

Reply:
Alright will set the default to false